playbooks/roles/ocp-config/tasks/main.yaml

```diff
@@ -101,6 +101,28 @@
     line: '\1mastersSchedulable: False'
     backrefs: yes
 
+- name: Chrony configuration
+  block:
```

Comment:
Suggestion would be to move full bock to a chrony.yaml task file. See squid.yaml for example.
Also, please mention the new variable in the role README.